python/sglang/srt/models/mixtral.py

```diff
@@ -380,5 +383,13 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                     )
                     weight_loader(param, loaded_weight)
 
+        if self.torchao_config:
+            param_suffixes = set(["proj.weight"])
+            param_suffixes.union(set(entry[0] for entry in stacked_params_mapping))
```

Comment:
why do we need `param_suffixes.union(set(entry[0] for entry in expert_params_mapping))` in qwen2_moe.py, but not here?

Reply:
makes sense, it seems everything ends with "proj.weight", so I'll just keep this